docs/apis/subsystems/xapi/index.md

```diff
@@ -0,0 +1,350 @@
+---
+title: xAPI (Experience API)
+tags:
+  - API
+  - core_xapi
+---
+
+## Overview
+
+This document contains a technical explanation on how the xAPI Moodle integration library works.
+
+The **Experience API (xAPI)** is an e-learning software specification that allows learning content and learning systems to speak to each other in a manner that records and tracks all types of learning experiences.
+
+xAPI defines web services that can be used by any plugin or APP to connect to the LMS in a standard way. The main actors in xAPI communication are:
+
+- **Client**: any plugin or APP that wants to use xAPI webservices
+- **xAPI LRS** (server site): responsible for storing and serve the data emitted by the clients
+
+One good aspect about xAPI is that the amount of messages that can be shared between the clients and the LRS is quite limited. The main message type is called **Statement** and all statements can be summarized as "*An actor XX executes action YY on object ZZ*".
+
+The typical statement object is a JSON element containing:
+
+- **Actor**: the person or group that do something
+- **Verb**: the action the Actor do
+- **Object**: the object on the Verb is executed. There are more than one type of objects that can be defined here but, for now, you can think about it as a specific "Quiz Attempt" for example.
+- Other fields: for now, the rest of xAPI fields have a basic data validation so every plugin is responsible for implementing it's own checks if they need them.
+The xAPI specification can be consulted in the xAPI specs page: https://github.com/adlnet/xAPI-Spec.
+
+## Moodle xAPI functionalities
+
+The final objective of Moodle xAPI library is NOT to implement a full LRS inside Moodle but to provide an easy way to handle xAPI statements within any plugin that needs it.
+
+For now, Moodle supports the main xAPI request which is called "**statement**". A statement must be seen as a tracking message that can be used by any plugin to store user activity directly to the Moodle without programming any additional web service.
+
+The current library implements:
+
+- A new webservice called **core_xapi_statement_post** to process xAPI statements and generate standard Moodle events.
+- A class **\core_xapi\handler** that any plugin can extend in order to user xAPI and generate specific plugin contexts. This way, any plugin could use xAPI without implementing any new web service.
+- A class **\core_xapi\iri** to easily translate random information into valid IRI values (needed for xAPI objects and verbs)
```

Comment:
What is an IRI value?

Reply:
I've replaced it with [IRI (Internationalized Resource Identifiers)](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#43-iris), to explain what IRI is.